bundesliga_app/views.py

```diff
@@ -8,6 +9,7 @@
 from bundesliga_app.models import Event
 from django.http import HttpResponseRedirect
 from .utils import get_auth_token
+from .models import MemberType, Discount
```

Comment:
**Just a tip !!** 
Eventbrite has a pattern to import more than 1 thing.
`from .models import (
    Discount,
    MemberType ,
)
`